zulipterminal/model.py

```diff
@@ -1392,7 +1392,8 @@ def _handle_typing_event(self, event: Event) -> None:
                     controller.show_typing_notification()
 
             elif event["op"] == "stop":
-                controller.active_conversation_info = {}
+                sender_email = self.user_dict[sender_email]["email"]
+                active_conversation_info.discard(sender_email)
```

Comment:
This commit doesn't yet add tests for this case, or the output the user sees.

Does the model change along show reasonable improvements? If so, you could add a test change for the model in this commit, and move the core changes into the next commit with the test you added.